pages/src/common/dropdown-menu.less

```diff
@@ -1,24 +1,45 @@
-.container-dropdown-menu {
-  width: 100%;
-  background-color: #343a40;
+// TODO: review after base styles creation
+// TODO: share with core ?
+.esl-sample-menu {
   display: flex;
+  width: 100%;
   height: 38px;
+  background-color: #343a40;
```

Comment:
It would be nice to have the color values in separate variables.

Reply:
Will be updated in bounds of redesign stories